Makefile

```diff
@@ -875,6 +875,21 @@ bench: bench-net bench-http bench-fs bench-tls
 
 bench-ci: bench
 
+mdlint-clean:
+	$(RM) -r tools/remark-cli/node_modules
+	$(RM) -r tools/remark-preset-lint-node/node_modules
+
+mdlint-build:
+	if [ ! -d tools/remark-cli/node_modules ]; then \
+		cd tools/remark-cli && ../../$(NODE) ../../$(NPM) install; fi
+	if [ ! -d tools/remark-preset-lint-node/node_modules ]; then \
+		cd tools/remark-preset-lint-node && ../../$(NODE) ../../$(NPM) install; fi
+
+mdlint: mdlint-build
+	@echo "Running Markdown linter..."
+	$(NODE) tools/remark-cli/cli.js -q -f \
+		./*.md doc src lib benchmark tools/doc/ tools/icu/
```

Comment:
We [lint code fragments](https://github.com/nodejs/node/blob/e88908d8684d4108a3079a6b6af0e5f4ce1b09b9/Makefile#L881) in these folders: `benchmark doc lib test tools`. Is there a reason why `test` folder is omitted here?